client.go

```diff
@@ -190,6 +190,26 @@ func (c *Client) trigger(channels []string, event string, data interface{}, sock
 	return unmarshalledBufferedEvents(response)
 }
 
+type BatchRequest struct {
+	Batch []Event `json:"batch"`
+}
+
+func (c *Client) TriggerBatch(batch []Event) (*BufferedEvents, error) {
```

Comment:
I guess the reason for using a new `Event` type here instead of having something similar to `Trigger` is to avoid receiving arrays of different lengths, and I guess the reason `Trigger` is not using an `Event` is for backwards compatibility... yet the inconsistency is a bit confusing. Also, the event name is called `event` in `Trigger` and `Name` in the `Event` type. Would it make sense to add a new function that has the same behavior as `Trigger` but is called with an `Event` instead? Then we could rename that new function to `Trigger` in the next major release.